buildspecs/archetype-native-image-test.yml

```diff
@@ -0,0 +1,62 @@
+version: 0.2
```

Comment:
Does anything happen when you run this buildspec, or do you execute target just to see that the app will start up?

Reply:
Basically this verifies that archetype works, the generated application can be compiled to a native image and the native image can be executed